package.json

```diff
@@ -29,6 +29,9 @@
     "body-parser": "^1.15.2",
     "express": "^4.14.0",
     "lodash": "^4.14.1",
+    "mongo": "^0.1.0",
+    "mongodb": "^2.2.5",
```

Comment:
I'm not 100% sure but I don't think these need to be added as dependencies:

"mongo": "^0.1.0",
"mongodb": "^2.2.5",

can you check if you remove these if the server still works?